playbooks/install-development-env.yml

```diff
@@ -0,0 +1,13 @@
+---
+
+- hosts: cluster
```

Comment:
We should add a [dev] target in hosts.ini that specifies the development machine(s) where the dev environment has to be installed. We don't need to install go, node & npm on all cluster nodes.

Reply:
Added dev target